Jenkinsfile

```diff
@@ -307,370 +307,7 @@ pipeline {
       }
     }
 
-    stage('Update version') {
```

Comment:
For a smaller diff, you wan simply use
```
      when {
        expression {
          return false
        }
      }
```
See https://github.com/nuxeo/nuxeo/pull/3870/commits/2bd2ee91d9c2e922996797b683ea15c809b6f66f.

Reply:
I did it inline since it's a temp modification that can't go unnoticed
```
     when {expression {return false}}
```